components/feature/addons/src/main/java/mozilla/components/feature/addons/update/AddonUpdater.kt

```diff
@@ -530,19 +585,27 @@ internal class AddonUpdaterWorker(
                             Result.retry()
                         }
                     }
+                    saveRequest(extensionId, status)
```

Comment:
I found the name confusing here. The way it's being implemented, the `Request` is really a `Result`, right? We're saving only after we have called `update` and once we know what the result was. If that's the case, I'd vote for renaming this to `UpdateResult` everywhere. `ResultStorage` instead of `RequestStorage` (and DAO etc.), because this is not storing the request to update, but the outcome.

Reply:
👍

Reply:
What about `UpdateAttempt`?